### ISSUE TITLE
Restricted scanned maps should be more obvious

### ISSUE THREAD

**Issue by escowles** [pulibrary/pulmap#705]
There is currently a small gray lock next to the title of restricted scanned maps, which is easy to overlook.  The lock should be more obvious (larger and/or more brightly colored), and the user should be given some indication that they can request access to the map.
